pom.xml

```diff
@@ -292,7 +292,8 @@
         <executions>
           <execution>
             <id>sign-artifacts</id>
-            <phase>verify</phase>
+            <!-- we only need to sign if we are deploying (publishing) -->
+            <phase>deploy</phase>
```

Comment:
Only sign in deploy. Nice because now you can run `mvn verify` locally and in PRs without worrying about GPG setup.